example/gtfs-rt-simple.jv

```diff
@@ -0,0 +1,106 @@
+
+    block GTFSRTVehiclePositionFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-vehicle-position";
+	}
+
+    block GTFSRTAlertFeedExtractor oftype HttpExtractor {
+		url: "https://proxy.transport.data.gouv.fr/resource/bibus-brest-gtfs-rt-alerts";
+	}
+
+    block GtfsRTTripUpdateInterpreter oftype GtfsRTInterpreter{
+        entity: "trip_update";
+    }
+
+    block GtfsRTAlertInterpreter oftype GtfsRTInterpreter{
+        entity: "alert";
+    }
+
+    block GtfsRTVehiclePositionInterpreter oftype GtfsRTInterpreter{
+        entity: "vehicle";
+    }
+    block TripUpdateTableInterpreter oftype TableInterpreter {
+		header: true;
+		columns:[
+			"header.gtfs_realtime_version" oftype text, 
+			"header.timestamp" oftype text, 
+            "header.incrementality" oftype text, 
+            "entity.id" oftype text, 
+            "entity.trip_update.trip.trip_id" oftype text, 
+            "entity.trip_update.trip.route_id" oftype text,
+            "entity.trip_update.stop_time_update.stop_sequence" oftype text, 
+            "entity.trip_update.stop_time_update.stop_id" oftype text,
+            "entity.trip_update.stop_time_update.arrival.time" oftype text, 
+            "entity.trip_update.stop_time_update.departure.time" oftype text,
+		];
+	}
+
+    block VehiclePositionTableInterpreter oftype TableInterpreter {
+		header: true;
+		columns:[  
+            "header.gtfs_realtime_version" oftype text,  
+            "header.timestamp" oftype text,  
+            "header.incrementality" oftype text,  
+            "entity.id" oftype text,  
+            "entity.vehicle_position.vehicle_descriptor.id" oftype text,  
+            "entity.vehicle_position.trip.trip_id" oftype text,  
+            "entity.vehicle_position.trip.route_id" oftype text,  
+            "entity.vehicle_position.position.latitude" oftype text, 
+            "entity.vehicle_position.position.longitude" oftype text, 
+            "entity.vehicle_position.timestamp" oftype text
+        ];
+	}
+
+    block AlertTableInterpreter oftype TableInterpreter {
+		header: true;
+		columns:[  
+            'header.gtfs_realtime_version' oftype text,
+            'header.timestamp' oftype text,
+            'header.incrementality'  oftype text,
+            'entity.id' oftype text,
+            'entity.alert.informed_entity.route_id' oftype text,
+            'entity.alert.header_text' oftype text,
+            'entity.alert.description_text' oftype text,
+        ];
+	}
+
+    block TripUpdateLoader oftype SQLiteLoader {
+		table: "gtfs-rt-trip_update";
+		file: "./gtfs.sqlite";
+        dropTable: false;
+	}
+
+    block VehicleLoader oftype SQLiteLoader {
+		table: "gtfs-rt-vehicle_position";
+		file: "./gtfs.sqlite";
+        dropTable: false;
+	}
+
+    block AlertLoader oftype SQLiteLoader {
+		table: "gtfs-rt-alert";
+		file: "./gtfs.sqlite";
+        dropTable: false;
+	}
+
+    GTFSRTTripUpdateFeedExtractor
```

Comment:
Style: I think all non-source blocks should be all indented only once.

Reply:
changed